### PR TITLE
rbd: make it more understandable when adding peer returns error

### PR DESCRIPTION
**Pull request by Songweibin**
When adding peer for a disabled pool, the error return seems hard to understand.
`[root@s247 ~]# rbd mirror pool peer add rbd client.admin@cluster ` 
`rbd: error adding mirror peer`  
`2017-07-13 14:22:42.287189 7f606e888bc0 -1 librbd::api::Mirror: peer_add: failed to add mirror peer` `'0x7ffdbce45dc0': (22) Invalid argument`  
And now, it looks like that: 
`[root@s248 ~]# rbd mirror pool peer add p1 client.admin@cluster `   
`rbd: error adding mirror peer`
`2017-07-13 14:52:22.740104 7f419822fbc0 -1 librbd::api::Mirror: peer_add: failed to add mirror peer: mirroring must be enabled on the pool`

Signed-off-by: songweibin <song.weibin@zte.com.cn>

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

**Comment by Songweibin** [comment]
@dillaman  Modified and updated as your suggestion, could you please have a look? Thanks.

File: src/tools/rbd/action/MirrorPool.cc [hunk 1]
```diff
@@ -528,6 +529,21 @@ int execute_peer_add(const po::variables_map &vm) {
   if (r < 0) {
     return r;
   }
+  
+  cls::rbd::MirrorMode mirror_mode;
+  r = librbd::cls_client::mirror_mode_get(&io_ctx, &mirror_mode);
```
Comment (dillaman) [inline src/tools/rbd/action/MirrorPool.cc]: Use the librbd public API -- ```librbd::RBD::mirror_mode_get```

**Comment by Songweibin** [comment]
@dillaman  done. thanks for your review.

**Pull request merged**